pennylane/transforms/defer_measurements.py

```diff
@@ -404,11 +404,10 @@ def _(
 
         return []
 
-    def defer_measurements_plxpr_to_plxpr(
-        jaxpr, consts, targs, tkwargs, *args
-    ):  # pylint: disable=unused-argument
+    def defer_measurements_plxpr_to_plxpr(jaxpr, consts, _, tkwargs, *args):
+        """Function for applying the ``defer_measurements`` transform on plxpr."""
 
-        if (aux_wires := tkwargs.get("aux_wires", None)) is None:
+        if tkwargs.get("aux_wires", None) is None:
```

Comment:
Personal preference (your call),
```suggestion
        if not tkwargs.get("aux_wires", None):
```